pkgs/misc/openrussian-cli/default.nix

```
@@ -37,8 +37,8 @@ stdenv.mkDerivation rec {
     cp openrussian $out/bin
 
     wrapProgram $out/bin/openrussian \
-      --prefix LUA_PATH ';' '${lua.pkgs.lib.genLuaPathAbsStr luaEnv}' \
-      --prefix LUA_CPATH ';' '${lua.pkgs.lib.genLuaCPathAbsStr luaEnv}'
+      --prefix LUA_PATH ';' '${lua.pkgs.luaLib.genLuaPathAbsStr luaEnv}' \
+      --prefix LUA_CPATH ';' '${lua.pkgs.luaLib.genLuaCPathAbsStr luaEnv}'
```

Could we do the following if added to inputs?

```suggestion
      --prefix LUA_CPATH ';' '${luaLib.genLuaCPathAbsStr luaEnv}'
```

---

`luaLib` isn't in the pkgs scope so it would have to be inherit (which I don't like)